### PR TITLE
added rump fast matrix multiplication

### PR DESCRIPTION
**Pull request by lucaferranti**

### CODE REVIEW

File: src/operations/mult.jl [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+"""
+    set_multiplication_mode(multype)
+
+Sets the algorithm used to perform matrix multiplication with interval matrices.
+
+### Input
+
+- `multype` -- symbol describing the algorithm used
+   - `:slow` -- uses traditional matrix multiplication algorithm.
+   - `:rank1` -- uses rank1 update
```
Comment (schillic) [inline src/operations/mult.jl:16]: I think `:rank1` should be removed (for now).

**Review by schillic — APPROVED** [review]

**Review by mforets — APPROVED** [review]

**Comment by mforets** [comment]
Excellent!

**Comment by mforets** [comment]
@lucaferranti  is this ready to merge?

**Comment by lucaferranti** [comment]
yes, should be ready!

**Pull request merged**